.github/workflows/spec_tests.yml

```diff
@@ -11,77 +11,6 @@ concurrency:
   cancel-in-progress: true
```

Comment:
While we're here, can we rename the name of this workflow from `Build Candlepin image and run spec tests` to something more concise? Like 'Run spec tests'? The fact that we're building/running container images is just an implementation detail

Reply:
Good catch! I agree on the name change. Updated to 'Run spec tests'